pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1,29 +1,30 @@
-{ config
-, lib
-, fetchurl
-, callPackage
-, vscode-utils
-, asciidoctor
-, nodePackages
-, python3Packages
-, jdk
-, llvmPackages
-, llvmPackages_14
-, nixpkgs-fmt
-, protobuf
-, jq
-, shellcheck
-, moreutils
-, racket
-, clojure-lsp
-, alejandra
-, millet
-, shfmt
-, typst-lsp
-, typst-preview
-, autoPatchelfHook
-, zlib
-, stdenv
+{
```

Comment:
I have used `sort` to order arguments automatically.
Reason is an expectation of automating checking alphabetical order in CI.
I suggest the use of `sort` to order arguments to avoid unnecessary conflicts from unstable.

Reply:
Sorry, not sure I fully understand, do you use sort somehow standalone or is it an option for nixfmt-rfc-style I haven't found?

Reply:
It's unrelated to `nixfmt`. I have manually ordered parameters by alphabetical order using `sort`. With the intention of using `sort` to automate checking the alphabetical order of parameters in CI. So, in `unstable ` at present moment, arguments are alphabetically ordered. If you only use `nixfmt` at stable, the issue of continued failing automatic back-ports will keep happening. Because the argument list in unstable is alphabetically ordered at present moment.

Reply:
You can make this alphabetical ordering a separate commit. And then, use nixfmt after.
Sorry about the extra work. And failing to communicate things properly/in advance.

Reply:
Understood, thanks for the input - should be solved now ✌🏻